project/AppDependencies.scala

```diff
@@ -19,10 +19,10 @@ object AppDependencies {
     // FORMAT: OFF
     "uk.gov.hmrc"            %% "bootstrap-test-play-30" % bootstrapVersion,
     "org.scalatest"          %% "scalatest"              % "3.2.19",
-    "org.jsoup"              % "jsoup"                   % "1.18.1",
+    "org.jsoup"              % "jsoup"                   % "1.18.3",
     "org.playframework"      %% "play-test"              % current,
```

Comment:
normally we do not need to explicitly add play-test dependency as it is provided by bootstrap, so can be removed here